### DISCUSSION TITLE
Showcases and Marketing

### DISCUSSION THREAD

**Discussion by crschnick** [mkpaz/atlantafx#37] [General]
I was looking through the available documentation but didn't stumble upon any list of practical showcases, i.e. real world applications. I think libraries like this need a showcase section to convince potential users that it is actually viable.

Furthermore, using the sampler overview screen for marketing purposes, i.e. displaying it on top of the readme, is contraproductive in my opinion. The sampler overview page gives of a bad first impression as there is no color and layout harmony (which is intended as the overview is meant to showcase individual components). The better stuff, which shows more than just individual components, is never shown in the screenshots and is listed for example at the bottom in the sampler application under the showcase section.

When I first saw AtlantaFX being advertised (I think on reddit) a while ago, I brushed over it as I wasn't very impressed with the shown screenshots (the sampler overview page). But now that I tried it out, it is much better than expected. So I would recommend to improve your marketing material, that shouldn't be too hard with the quality of the sampler and library.

**Comment by mkpaz** [comment]
Thanks. I somewhat agree with that. Though placing components on the demo page is pretty much a common practice. Overview page just need to be improved. I'll add this to my backlog.

**avalonia**

![image](https://user-images.githubusercontent.com/25181998/218939280-91ca0090-d84c-4689-b455-e21a1f68d8f5.png)

**gtk4**

![image](https://user-images.githubusercontent.com/25181998/218939411-d101d6ce-bad4-410e-8076-11b07a101be0.png)

**Comment by crschnick** [comment]
I guess the difference is that almost everyone knows these projects and has probably seen countless other projects using them. At least avalonia has several showcases: https://github.com/AvaloniaUI/Avalonia#showcase

So why not put the 5-10 best looking pages into the readme, documentation, and advertisements on reddit in addition to the overview page. And maybe compile a list of the best looking applications out there that use AtlantaFX.

**Comment by crschnick** [comment]
I see that the readme has been updated and it looks very impressive now in my opinion!
